src/query/interpreter.cpp

```diff
@@ -791,6 +791,33 @@ Callback HandleStreamQuery(StreamQuery *stream_query, const Parameters &paramete
   }
 }
 
+Callback HandleConfigQuery() {
+  Callback callback;
+  callback.header = {"name", "default_value", "current_value"};
```

Comment:
I think `SHOW CONFIG` printing both the default and current config is redundant. A couple of options to consider: 

1. Introduce `SHOW DEFAULT CONFIG` which basically prints the default values
2. Remove the `default_value` completely
3. Keep things as is. 

Any comments on this @gitbuda and @antaljanosbenjamin?

Reply:
Just to add to this question, if modified, is there anything else we would like to add?

- std::string name;            // the name of the flag
- std::string type;            // the type of the flag: int32, etc
- std::string description;     // the "help text" associated with the flag
- std::string current_value;   // the current value, as a string
- std::string default_value;   // the default value, as a string
- std::string filename;        // 'cleaned' version of filename holding the flag

description might make sense.

Reply:
@kostasrim I think showing the default value is okay, it is something people might not know from top of their heart.

@gvolfing I think the description makes sense to output. Another thing to consider (it is just an idea), is to format the flags into `--lower-dash-case-format` instead of `current_sneak_case`. I think users might happen to copy these flags and it is easy to miss the difference between `_` and `-`.

Reply:
The returned value of the first column is the _name_ of the given flag not the flag itself.
`snake_case_flag` is the way they are used inside the `std::string`s of gFlag's `CommandLineFlagInfo` and in other places such as `memgraph/config/flags.yaml`.
That being said, I can change it if you insist.

Reply:
I wasn't aware of that, haven't used the flag files more than once or two. I think based on what you said, it is okay to have them in `sneak_case`.